README.md

```diff
@@ -82,6 +82,21 @@ events.broadcast('post.created', user_id: 1)
 # => 'something created'
 ```
 
+### Formatters
+You can use different formatters for displaying information about all your events for event instance. Now hanami-events support:
```

Comment:
Should we mention that it returns just list of registered events?
_You can use different formatters for displaying list of registered events for event instance_